css/custom.css

```diff
@@ -6,6 +6,20 @@
     top: 0;
   }
 }
+
+
+#uniformlink:hover
```

Comment:
You are giving ID to all the links ? Id's are unique they shouldnt be duplicated.

directly use 
```css
a { 
   /* Your css tules */
}```